addon/templates/components/validated-input.hbs

```diff
@@ -38,18 +38,19 @@
   {{#if (eq type "select")}}
 
     {{one-way-select
-      value            = (or value (get model name))
-      options          = options
-      optionLabelPath  = optionLabelPath
-      optionValuePath  = optionValuePath
-      optionTargetPath = optionTargetPath
-      name             = name
-      class            = config.css.control
-      update           = (action "update")
-      focusOut         = (action "setDirty")
-      includeBlank     = includeBlank
-      disabled         = disabled
-      multiple         = multiple
+    value               = (or value (get model name))
+    options             = options
+    optionLabelPath     = optionLabelPath
+    optionValuePath     = optionValuePath
+    optionTargetPath    = optionTargetPath
+    name                = name
+    class               = "form-control"
+    update              = (action "update")
+    focusOut            = (action "setDirty")
+    prompt              = (if includeBlank (or includeBlank null))
+    disabled            = disabled
+    multiple            = multiple
+    promptIsSelectable  = (if promptIsSelectable (or promptIsSelectable false))
```

Comment:
Isn't `promptIsSelectable = promptIsSelectable` also backwards compatible? `false` is the default and we didn't support this option before.